cfgov/v1/models/base.py

```diff
@@ -217,6 +223,41 @@ def specific_categories_query(block, parent_slug):
         return {search_type: queryset for search_type, queryset in
                 related.items() if queryset}
 
+    def flush(self):
+        if not settings.ENABLE_AKAMAI_CACHE_PURGE:
+            logger.info(
+                'Page {slug} (id = {id}) was not invalidated because '
+                'ENABLE_AKAMAI_CACHE_PURGE is not set'.format(
+                    slug=self.slug,
+                    id=self.id
+                )
+            )
+            return
+        auth, fast_purge_url = get_akamai_credentials()
+        headers = {'content-type': 'application/json'}
+        payload = {
+            'action': 'invalidate',
+            'hostname': settings.WAGTAIL_SITE_NAME,
+            'objects': [
+                self.relative_url(self.get_site())
+            ]
+        }
+        resp = requests.post(
+            fast_purge_url,
+            headers=headers,
+            data=json.dumps(payload),
+            auth=auth
+        )
+        logger.info(
+            'Attempted to invalidate page {slug} (id = {id}), '
```

Comment:
Okay I added a bit of logging here and above.  On a failure (that I happen to be getting now because of some config thing) this would look something like:
```
Attempted to invalidate page faqs (id = 993), got back response {"supportId": "17PY148XXXXX", "title": "unauthorized arl", "httpStatus": 403, "detail": "http://consumerfinance.gov/foia-requests/faqs/", "describedBy": "https://api.ccu.akamai.com/ccu/v2/errors/unauthorized-arl"}
```
I have it set to log this regardless of if the request is successful or not, since it might be useful in the beginning to log what we are actually successfully flushing.  And the page will error out so we'll know immediately if something's not working.